stable/elasticsearch-exporter/values.yaml

```diff
@@ -53,6 +53,12 @@ es:
   ##
   timeout: 30s
 
+  ## Skip SSL verification when connecting to Elasticsearch
+  ## (only available if image.tag >= 1.0.4rc1)
+  ##
+  sslSkipVerify: false
```

Comment:
can you please add the new argument to the configuration sections of the README? (https://github.com/helm/charts/blob/master/stable/elasticsearch-exporter/README.md#configuration)

Reply:
done